### DISCUSSION TITLE
How do I download Install libusb0 (v1.2.6.0)

### DISCUSSION THREAD

**Discussion by Kaptajnen9325** [opentrack/opentrack#1411] [Q&A]
Install libusb0 (v1.2.6.0)
Headline says it all...can someone explain this for a total non tech?

**Comment by GO63-samara** [comment]
https://github.com/opentrack/opentrack/issues/1383#issuecomment-1013661144

**Comment by GO63-samara** [comment]
https://github.com/opentrack/opentrack/issues/1135

**Comment by sthalik** [comment]
https://github.com/opentrack/opentrack/wiki/PS3-Eye-open-driver-instructions
